src/io/flutter/sdk/FlutterSdkManager.java

```diff
@@ -20,26 +21,20 @@
  * Monitors the application library table to notify clients when Flutter SDK configuration changes.
  */
 public class FlutterSdkManager {
-
-  private static FlutterSdkManager INSTANCE;
   private final EventDispatcher<Listener> myDispatcher = EventDispatcher.create(Listener.class);
   private final LibraryTableListener myLibraryTableListener = new LibraryTableListener();
   private boolean isFlutterConfigured;
 
+  public static FlutterSdkManager getInstance() {
```

Comment:
Are we expecting this to be non-null at call sites (or do we want to be able to?)  If so, maybe add `@NotNull`?